src/Drupal/Commands/field/FieldBaseInfoCommands.php

```diff
@@ -34,40 +35,36 @@ public function __construct(
     /**
      * List all base fields of an entity type
      *
-     * @command field:base-info
-     * @aliases field-base-info,fbi
-     *
-     * @param string $entityType
-     *      The machine name of the entity type
-     *
-     * @option show-machine-names
-     *      Show machine names instead of labels in option lists.
-     *
-     * @default-fields field_name,required,field_type,cardinality
-     * @field-labels
-     *      label: Label
-     *      description: Description
-     *      field_name: Field name
-     *      field_type: Field type
-     *      required: Required
-     *      translatable: Translatable
-     *      cardinality: Cardinality
-     *      default_value: Default value
-     *      default_value_callback: Default value callback
-     *      allowed_values: Allowed values
-     *      allowed_values_function: Allowed values function
-     *      handler: Selection handler
-     *      target_bundles: Target bundles
-     * @filter-default-field field_name
      * @table-style default
-     *
-     * @usage drush field:base-info taxonomy_term
-     *      List all base fields.
-     * @usage drush field:base-info
-     *      List all base fields and fill in the remaining information through prompts.
-     *
-     * @version 11.0
      */
+    #[CLI\Command(name: 'field:base-info', aliases: ['field-base-info', 'fbi'])]
+    #[CLI\Argument(name: 'entityType', description: 'The machine name of the entity type.')]
+    #[CLI\Option(name: 'show-machine-names', description: 'Show machine names instead of labels in option lists.')]
+    #[CLI\DefaultFields(fields: [
```

Comment:
We are switching to DefaultTableFields instead which still shows all fields in non table output formats.